### DISCUSSION TITLE
schematics-utilities should be used as dev-dependencies

### DISCUSSION THREAD

**Discussion by bifty** [salesforce/akita#756] [General]
## I'm having a problem regarding the used dependencies -> licenses

[X] Bad Dependency Management
</code></pre>

## Current behavior

**schematics-utilities** are used as dependency.
https://github.com/datorama/akita/blob/cac039bfc2f78f9189f2315fda3ed5cecc3a3dc3/package.json#L81

## Expected behavior

To reduces license problems when using akita it should be implemented as dev-dependency if possible.

## What is the motivation / use case for changing the behavior?
- Reduce license issues and dependencies in production.
- critical for some companies to use akita this way

**Comment by NetanelBasal** [comment]
It's not a dev dependency. You need it to use schematics.
